### PR TITLE
Translate of benchmarks.md to Spanish

### PR DESCRIPTION
**Pull request by wfpinedar**
First try to contribute to FastaAPI with translations

### CODE REVIEW

File: docs/es/docs/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+La jerarquia es algo como:
+
+* **Uvicorn**: un servidor ASGI (Asynchronous Server Gateway Interface)
+  * **Starlette**: (usa Uvicorn) un microframework web
+    * **FastAPI**: (usa Starlette) un API microframework con varias funcionaldades adicionales para la creación de API, con validación de datos, etc.
+
+* **Uvicorn**:
+  * Tendrá el mejor rendimiento, ya que no tiene mucho código adicional aparte del servidor en sí.
+  * No escribirías una aplicación en Uvicorn directamente. Eso significaría que su código tendría que incluir más o menos, al menos, todo el código proporcionado por Starlette (o ** FastAPI **). Y si hiciera eso, su aplicación final tendría la misma sobrecarga que haber usado un framework y minimizando el código y los errores de su aplicación.
```
Comment (Frans06) [inline docs/es/docs/benchmarks.md:24]:
```suggestion
  * No escribirías una aplicación en Uvicorn directamente. Eso significaría que tu código tendría que incluir más o menos, al menos, todo el código proporcionado por Starlette (o ** FastAPI **). Y si hiciera eso, su aplicación final tendría la misma sobrecarga que haber usado un framework y minimizando el código y los errores de su aplicación.
```

File: docs/es/docs/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  * Si estas comparando Uvicorn, compáralo con Daphne, Hypercorn, uWSGI, etc. Servidores de aplicaciones.
+* **Starlette**:
+  * Tendrá el siguiente mejor desempeño, después de Uvicorn. De hecho, Starlette usa Uvicorn para ejecutarse. Por lo tanto, probablemente solo podria volverse "más lento" que Uvicorn al tener que ejecutar más código.
```
Comment (Frans06) [inline docs/es/docs/benchmarks.md:27]:
```suggestion
  * Tendrá el siguiente mejor desempeño, después de Uvicorn. De hecho, Starlette usa Uvicorn para ejecutarse. Por lo tanto, probablemente solo se volvería "más lento" que Uvicorn cuando tenga que ejecutar más código.
```

File: docs/es/docs/benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  * Pero este provee las herramientos para contruir aplicaciones web simples, con direccionamiento basadao en rutas, etc.
+  * Si esta comparando Starlette, compare este con Sanic, Flask, Django, etc. frameworks web (o microframeworks).
+* **FastAPI**:
+  * De la misma manera que Starlette usa Uvicorn y no puede ser más rápido que él, **FastAPI** usa Starlette, entonces no puede ser más rápido que él.
+  * FastAPI proporciona más funciones además de Starlette. Funciones que casi siempre se necesitan al crear APIs, como validación y serialización de datos. Y al usarlo, obtiene documentación automática de forma gratuita (la documentación automática ni siquiera agrega gastos generales a las aplicaciones en ejecución, se genera al inicio).
```
Comment (Frans06) [inline docs/es/docs/benchmarks.md:32]:
```suggestion
  * FastAPI proporciona más funciones además de Starlette. Funciones que casi siempre se necesitan al crear APIs, como validación y serialización de datos. Y al usarlo, obtienes documentación automática gratis (la documentación automática ni siquiera agrega sobre carga a las aplicaciones en ejecución, esta se genera al inicio).
```

File: docs/es/docs/benchmarks.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  * Si no usó FastAPI y usó Starlette directamente (u otra herramienta, como Sanic, Flask, Responder, etc.), tendría que implementar toda la validación y serialización de datos usted mismo. Por lo tanto, su aplicación final aún tendría la misma sobrecarga que si se hubiera creado con FastAPI. Y en muchos casos, esta validación y serialización de datos es la mayor cantidad de código escrito en las aplicaciones.
```
Comment (Frans06) [inline docs/es/docs/benchmarks.md:33]:
```suggestion
  * Si no usaste FastAPI y usaste Starlette directamente (u otra herramienta, como Sanic, Flask, Responder, etc.), tendrías que implementar toda la validación y serialización de datos tú mismo. Por lo tanto, tu aplicación final aún tendría la misma sobre carga que si se hubiera creado con FastAPI. Y en muchos casos, esta validación y serialización de datos es la mayor cantidad de código escrito en las aplicaciones.
```

File: docs/es/docs/benchmarks.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  * Entonces, usando FastAPI usted esta ahorrando tiempo de desarrollo, errores, lineas de codigo y probablemente obtendria el mismo desempeño (o mejor) que si usted no usare este (ya que tendría que implementarlo todo en su código ).
```
Comment (Frans06) [inline docs/es/docs/benchmarks.md:34]:
```suggestion
  * Entonces, usando FastAPI estas ahorrando tiempo de desarrollo, errores, lineas de código y probablemente obtendrias el mismo desempeño (o mejor) que si no usaras este (ya que tendría que implementarlo todo en su código ).
```

File: docs/es/docs/benchmarks.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  * Si está comparando FastAPI, compárelo con un framework de aplicación web (o un conjunto de herramientas) que proporciona validación de datos, serialización y documentación, como Flask-apispec, NestJS, Molten, etc. Marcos con validación de datos automática integrada, serialización y documentación.
```
Comment (Frans06) [inline docs/es/docs/benchmarks.md:35]:
```suggestion
  * Si está comparando FastAPI, compárelo con un framework de aplicación web (o un conjunto de herramientas) que proporciona validación de datos, serialización y documentación, como Flask-apispec, NestJS, Molten, etc. Frameworks con validación de datos automática, serialización y documentación, integrados.
```

**Review by Frans06 — CHANGES_REQUESTED** [review]

**Comment by Frans06** [comment]
@wfpinedar  I'm adding some suggestions. Most of it because on English there is not "Honorificos". usted ~= tu. I think it could be more standard. One to other misspelling and grammar. In any case great translation.  Don't forget to add section on mkdocs.yml.
```
site_name: FastAPI
# More stuff
nav:
- FastAPI: index.md
- Languages:
  - en: /
  - es: /es/
- benchmark.md
```

File: docs/es/docs/benchmarks.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+Pero cuando verificas los puntos de referencia y las comparaciones, deberias tener en mente lo siguiente.
+
+## Puntos de referencia y velocidad
+
+Cuando verificas las referencias,  
+When you check the benchmarks, es común ver varias herramientas de diferentes tipos comparadas como equivalentes.
+
+Específicamente, al ver Uvicorn, Starlette y FastAPI comparados juntos (entre muchas otras herramientas).
+
+Cuanto más simple sea el problema resuelto por la herramienta, mejor rendimiento obtendrá. Y la mayoría de los puntos de referencia no prueban las funciones adicionales proporcionadas por la herramienta.
```
[the remark this answers is not in the document]
Comment (wfpinedar) [reply docs/es/docs/benchmarks.md]: Solve in: 28a89ca56c0a50353d9ce4b1fcc0453ee512d3f2

**Review by Frans06 — APPROVED** [review]

File: docs/es/docs/benchmarks.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  * Pero este provee las herramientos para contruir aplicaciones web simples, con direccionamiento basadao en rutas, etc.
+  * Si esta comparando Starlette, compare este con Sanic, Flask, Django, etc. frameworks web (o microframeworks).
+* **FastAPI**:
+  * De la misma manera que Starlette usa Uvicorn y no puede ser más rápido que él, **FastAPI** usa Starlette, entonces no puede ser más rápido que él.
+  * FastAPI proporciona más funciones además de Starlette. Funciones que casi siempre se necesitan al crear APIs, como validación y serialización de datos. Y al usarlo, obtienes documentación automática gratis (la documentación automática ni siquiera agrega sobre carga a las aplicaciones en ejecución, esta se genera al inicio).
+  * Si no usaste FastAPI y usaste Starlette directamente (u otra herramienta, como Sanic, Flask, Responder, etc.), tendrías que implementar toda la validación y serialización de datos tú mismo. Por lo tanto, tu aplicación final aún tendría la misma sobre carga que si se hubiera creado con FastAPI. Y en muchos casos, esta validación y serialización de datos es la mayor cantidad de código escrito en las aplicaciones.
```
Comment (pedabraham) [inline docs/es/docs/benchmarks.md:32]:
```suggestion
  * Si no usaste FastAPI y usaste Starlette directamente (u otra herramienta, como Sanic, Flask, Responder, etc.), tendrías que implementar toda la validación y serialización de datos tú mismo. Por lo tanto, tu aplicación final aún tendría la misma sobrecarga que si se hubiera creado con FastAPI. Y en muchos casos, esta validación y serialización de datos es la mayor cantidad de código escrito en las aplicaciones.
```

**Review by pedabraham — COMMENTED** [review]
Hi! I found some typos.
Oye, Encontré unas faltas de ortografía.

File: docs/es/docs/benchmarks.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
+# Puntos de referencia
+
+Los puntos de referencia  independientes, TechEmpower, colocan a las apliaciones **FastAPI** que se ejecutan bajo Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank"> uno de los frameworks de python mas rapidos disponibles</a>, solo por debajo de Starlette y Uvicorn (utilizados internamente por FastAPI). (*)
```
Comment (israteneda) [inline docs/es/docs/benchmarks.md:3]:
```suggestion
Los puntos de referencia  independientes, TechEmpower, colocan a las aplicaciones **FastAPI** que se ejecutan bajo Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank"> uno de los frameworks de python más rapidos disponibles</a>, solo por debajo de Starlette y Uvicorn (utilizados internamente por FastAPI). (*)
```

File: docs/es/docs/benchmarks.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+## Puntos de referencia y rapidez
+
+Cuando verificas las referencias, es común ver varias herramientas de diferentes tipos comparadas como equivalentes.
+
+Específicamente, al ver Uvicorn, Starlette y FastAPI comparados juntos (entre muchas otras herramientas).
+
+Cuanto más simple sea la solucion a un problema por parte de la herramienta, mejor rendimiento obtendrá. Y la mayoría de los puntos de referencia no prueban las funciones adicionales proporcionadas por la herramienta.
+
+La jerarquia es algo como:
```
Comment (israteneda) [inline docs/es/docs/benchmarks.md:15]:
```suggestion
La jerarquía es algo como:
```

File: docs/es/docs/benchmarks.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
+# Puntos de referencia
+
+Los puntos de referencia  independientes, TechEmpower, colocan a las apliaciones **FastAPI** que se ejecutan bajo Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank"> uno de los frameworks de python mas rapidos disponibles</a>, solo por debajo de Starlette y Uvicorn (utilizados internamente por FastAPI). (*)
+
+Pero cuando verificas los puntos de referencia y las comparaciones, deberias tener en mente lo siguiente.
```
Comment (israteneda) [inline docs/es/docs/benchmarks.md:5]:
```suggestion
Pero cuando verificas los puntos de referencia y las comparaciones, deberías tener en mente lo siguiente.
```

File: docs/es/docs/benchmarks.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+Específicamente, al ver Uvicorn, Starlette y FastAPI comparados juntos (entre muchas otras herramientas).
+
+Cuanto más simple sea la solucion a un problema por parte de la herramienta, mejor rendimiento obtendrá. Y la mayoría de los puntos de referencia no prueban las funciones adicionales proporcionadas por la herramienta.
+
+La jerarquia es algo como:
+
+* **Uvicorn**: un servidor ASGI (Asynchronous Server Gateway Interface)
+  * **Starlette**: (usa Uvicorn) un microframework web
+    * **FastAPI**: (usa Starlette) un API microframework con varias funcionaldades adicionales para la creación de APIs, con validación de datos, etc.
```
Comment (israteneda) [inline docs/es/docs/benchmarks.md:19]:
```suggestion
    * **FastAPI**: (usa Starlette) un API microframework con varias funcionalidades adicionales para la creación de APIs, con validación de datos, etc.
```

File: docs/es/docs/benchmarks.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+
+* **Uvicorn**:
+  * Tendrá el mejor rendimiento, ya que no tiene mucho código adicional aparte del servidor en sí.
+  * No escribirías una aplicación en Uvicorn directamente. Eso significaría que tu código tendría que incluir más o menos, al menos, todo el código proporcionado por Starlette (o ** FastAPI **). Y si hiciera eso, su aplicación final tendría la misma sobrecarga que haber usado un framework y minimizando el código y los errores de su aplicación.
+  * Si estas comparando Uvicorn, compáralo con Daphne, Hypercorn, uWSGI, etc. Servidores de aplicaciones.
+* **Starlette**:
+  * Tendrá el siguiente mejor desempeño, después de Uvicorn. De hecho, Starlette usa Uvicorn para ejecutarse. Por lo tanto, probablemente solo se volvería "más lento" que Uvicorn cuando tenga que ejecutar más código.
+  * Pero este provee las herramientos para contruir aplicaciones web simples, con direccionamiento basadao en rutas, etc.
```
Comment (israteneda) [inline docs/es/docs/benchmarks.md:27]:
```suggestion
  * Pero este provee las herramientas para construir aplicaciones web simples, con direccionamiento basado en rutas, etc.
```

**Review by israteneda — CHANGES_REQUESTED** [review]
Algunas pequeñas correcciones

File: docs/es/docs/benchmarks.md (hunk 11 above)
Comment (wfpinedar) [reply docs/es/docs/benchmarks.md]: solved in @90181d8

**Review by israteneda — APPROVED** [review]
LGTM ✅

**Comment by alejsdev** [comment]
This one was covered in a more recent PR (https://github.com/tiangolo/fastapi/pull/1353), but thanks for the effort @wfpinedar and thanks to everyone for the help 🤓

**Pull request closed without merging**